test/selector.spec.ts

```diff
@@ -259,4 +260,26 @@ describe('test selection paths', () => {
   it('filter notation should return a reference to the object', () => {
     expect(select('favorites.films[?(action=Rogue One)]', testData)).to.eq(testData.favorites.films);
   });
+
+  it('filter notation should return a reference to the object', () => {
+    expect(select('favorites.films[?(action=Rogue One)]', testData)).to.eq(testData.favorites.films);
+  });
```

Comment:
Nit: Looks like this test case was duplicated somehow (exists immediately above).

Reply:
Yep, ty; appears that I copied/pasted intending to use and then didn't :(